stanovy.tex

```diff
@@ -89,15 +89,21 @@ \section{Členství}
     o naplňování poslání Spolku.
     \item Ukončení členství:
     \begin{enumerate}[itemsep=0pt,topsep=0pt]
-        \item dobrovolné vystoupení člena – členství končí dnem doručení 
-        písemného oznámení o ukončení členství výboru,
+        \item dobrovolným vystoupení člena – členství končí dnem doručení 
+        oznámení o ukončení členství výboru,
         \item úmrtím člena,
         \item vyloučením člena – členství končí dnem doručení rozhodnutí 
         o vyloučení člena ze Spolku; výbor Spolku má právo vyloučit člena, 
         pokud svým jednáním porušuje zásady a cíle Spolku, zanedbává členské 
         povinnosti, ohrožuje řádně plnění poslání Spolku nebo se již déle než 
         rok nepodílí na naplňování poslání Spolku.
     \end{enumerate}
+    \item Změna typu členství:
+    \begin{enumerate}[itemsep=0pt,topsep=0pt]
+        \item žádostí člena - typ členství je změněn schválením výborem Spolku,
+        \item žádostí výboru Spolku - typ členství je změněn souhlasem dotčeného člena nebo marným uplynutím třicetidenní lhůty po dni doručení žádosti,
+        \item rozhodnutím výboru Spolku v případě, že jde o člena s aktivním členstvím, který se po delší dobu nepodílí na naplňování poslání Spolku.
+    \end{enumerate}
```

Comment:
Explicitní vyjasnění jak se členství mění, které zahrnuje požadavek práva pro výbor měnit členství na sympatizující se souhlasem i bez souhlasu a to i pouhým rozhodnutím...

Bod 3. je spíše pojistka, proti případům kdyby člen nesouhlasil, ale nebyl aktivní. Interpretace "delší" ponechávám plně na výboru.